ftd.js

```diff
@@ -1118,24 +1119,47 @@ window.ftd.post_init = function () {
         console_log("last_device", last_device);
     }
 
-    // function update_markdown_colors() {
-    //     // ftd#link-code.dark
-    //     // remove all colors from ftd.css: copy every deleted stuff in this function
-    //     var sheet = document.createElement('style')
-    //     sheet.innerHTML = """
-    //      .ft_md a:visited code {
-    //         color: window.ftd.get_value("main", "ftd#link-code.light");
-    //     }
-    //         body.fpm-dark .ft_md a:visited code {
-    //                 color: window.ftd.get_value("main", "ftd#link-code.dark");
-    //             }
-    //
-    //             body.fpm-dark .ft_md code {
-    //                 color: window.ftd.get_value("main", FTD_DARK_A__CODE);;
-    //             }
-    //     """;
-    //     document.body.appendChild(sheet);
-    // }
+    function update_markdown_colors() {
+        // remove all colors from ftd.css: copy every deleted stuff in this function
+        let markdown_style_sheet = document.createElement('style');
+
+        markdown_style_sheet.innerHTML = `
```

Comment:
In what version of JS was this syntax (back tic) added?

Reply:
[ECMAScript 6 // ECMAScript 2015](https://stackoverflow.com/questions/27678052/usage-of-the-backtick-character-in-javascript#:~:text=Summary%3A,also%20named%20template%20string%20literal.)